mw-webapp/src/logic/reportsTable/renderCellValue/helpers/updateJobDoneTime.ts

```diff
@@ -0,0 +1,14 @@
+import {JobDoneDAL} from "src/dataAccessLogic/JobDoneDAL";
+import {JobDone} from "src/model/businessModel/JobDone";
+
+/**
+ * Update JobDone
+ */
+export const updateJobDoneTime = async (time: string, uuid: string) => {
```

Comment:
time should bew number (+ add description)

Reply:
Changed